src/client/Microsoft.Identity.Client/Internal/Logger/LoggerHelper.cs

```diff
@@ -110,6 +113,7 @@ internal static string GetPiiScrubbedExceptionDetails(Exception ex)
                 {
                     sb.AppendLine(string.Format(CultureInfo.InvariantCulture, "HTTP StatusCode {0}", msalServiceException.StatusCode));
                     sb.AppendLine(string.Format(CultureInfo.InvariantCulture, "CorrelationId {0}", msalServiceException.CorrelationId));
+                    sb.AppendLine(string.Format(CultureInfo.InvariantCulture, "AAD Error Code AADSTS{0}", string.Join(" ", msalServiceException.ErrorCodes)));
```

Comment:
@trwalke we should probably start calling this "Microsoft Entra ID" instead of AAD 😀

Reply:
will do